index.js

```diff
@@ -241,10 +201,7 @@ function resolve_array(arr) {
  * @return {String}
  */
 
-function type(v) {
-  return isArray(v)
-    ? 'array'
-    : v.vo
-    ? 'vo'
-    : typeof v;
+function typeOf(v) {
```

Comment:
For the purpose of differentiating between arrays, functions and objects, what is the benefit of this over typeof?